packages/core/src/components/validation/validation.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../helpers';
+
+@mixin Validation() {
+  .ods-validation {
```

Comment:
My issue with the name is that it's always a red/negative/error message, even according to its own style.

I'd call it `error` instead, or something that denotes an unsuccessful/invalid state of validation.

Like, you wouldn't use this component for any other kind of message, for example:

```tsx
<Validation success>Your password complexity is safe enough!</Validation>
```

Reply:
I understand this.

However, component naming should still align both in design and engineering. I will suggest this change to designers.